dimagi/pages/templates/sections/sectors/maternal_and_child_health/overview.html

```diff
@@ -0,0 +1,54 @@
+{% load i18n %}
+{% load dimagidotcom %}
+
+<section class=""
+         data-lazybg="true"
+         id="overview">
+  <div class="content pad-top-medium pad-bottom-medium">
+    <div class="pad-bottom-line pad-bottom-small-at-medium pad-bottom-small-at-large pad-bottom-small-at-extra-large">
+      <strong>
+        {% blocktrans %}
+          MOBILE DATA COLLECTION
+        {% endblocktrans %}
+      </strong>
+      <h2 class="">
+        {% blocktrans %}
+          Maternal & Child Healthcare
+        {% endblocktrans %}
+      </h2>
+    </div>
+    <div class="grid grid-2-at-medium grid-1-at-small grid-row-equal-heights ">
+      <div class="column">
+        <p>
+          {% blocktrans %}
+            Mobile data collection has been proven to aid the maternal and child health sector. 
+            Research has shown it to increase antenatal care by 41% and improve facility delivery rates by over 100%. 
+            Programs featuring mobile data collection have also seen family members participate more in antenatal care, with husbands engaging 60% 
+            and mothers-in-law 110% more often than in paper-based programs.
+          {% endblocktrans %}
+        </p>
+        <p>
+          {% blocktrans %}
+            Organizations are using mobile data collection platform to <a href="https://www.youtube.com/watch?v=HRjJFuuog0g&list=PLLIP1SB1dlQL3VuOzOcAN76tx3_k-nCjR&index=7" target="_blank">digitize the partograph</a> 
+            and ICMI to allow programs to keep track of pregnancies and
+            clinic visits, diagnose any issues from data collected, and offer advice to mothers and their families on what to expect faster and 
+            more accurately than ever before.
+          {% endblocktrans %}
+        </p>
+        <p>
+          {% blocktrans %}
+            More projects on CommCare are focused on maternal and child health than any other sector. 
+            In fact, our platform registers one in every 110 pregnancies worldwide.
+          {% endblocktrans %}
+        </p>              
+      </div>{# .column #}
+      <div class="column" id="ag-intro">
+        <div class="">
```

Comment:
i'm going to stop marking the empty class attributes. please just get rid of all of them